README.md

```diff
@@ -2,106 +2,119 @@
-
-# Parse API files, render templates to src/lib folder and copy static files to src/lib
-$ yarn run renderTemplates
+- Node `^12.x`, `^14.x`, or `^16.x`
 
-# Launch sample application
-$ yarn start
+### Installation
 
-# Transpile and minify
-$ yarn build:lib
-
-# Run tests
-$ yarn test
+```bash
+npm install commerce-sdk-isomorphic
 ```
 
-## Usage 
-
-An example React App is available at `./src/environment/App` directory. To use the sample application, configure these parameters in `./src/environment/config.js` file.
+### Usage
 
 > **Note:** These are required parameters.
 
-| Parameter      | Description                                                                                                                             |
-| -------------- | :-------------------------------------------------------------------------------------------------------------------------------------- |
-| clientId       | ID of the client account created with Salesforce Commerce.                                                                              |
-| organizationId | The unique identifier for your Salesforce identity.                                                                                     |
-| shortCode      | Region specific merchant ID.                                                                                                            |
-| siteId         | A unique site ID (for example, RefArch or SiteGenesis).                                                                                 |
+| Parameter      | Description                                                                |
+| -------------- | :------------------------------------------------------------------------- |
+| clientId       | ID of the client account created with Salesforce Commerce.                 |
+| organizationId | The unique identifier for your Salesforce identity.                        |
+| shortCode      | Region-specific merchant ID.                                               |
+| siteId         | Name of the site to access data from, for example, RefArch or SiteGenesis. |
 
 ```javascript
 /**
  * Configure required parameters
- * 
+ *
  * To learn more about the parameters please refer to https://developer.commercecloud.com/s/article/CommerceAPI-Get-Started
  */
+import {helpers, ShopperLogin, ShopperSearch} from 'commerce-sdk-isomorphic';
+
 // Create a configuration to use when creating API clients
 const config = {
-    proxy: 'https://localhost:3000',
-    headers: {},
-    parameters: {
-      clientId: '<your-client-id>',
-      organizationId: '<your-org-id>',
-      shortCode: '<your-short-code>',
-      siteId: '<your-site-id>'
-    }
-}
+  proxy: 'https://localhost:3000', // Routes API calls through a proxy when set
+  headers: {},
+  parameters: {
+    clientId: '<your-client-id>',
+    organizationId: '<your-org-id>',
+    shortCode: '<your-short-code>',
+    siteId: '<your-site-id>',
+  },
+  throwOnBadResponse: true,
```

Comment:
We want to throw on bad responses as a default configuration?

Reply:
We don't have it as default in the SDK because it would be a breaking change. Ideally, it would have been the default behavior from the start, so I'm adding it in this example since it is what nearly everyone wants.